requirements/static/py3.5/linux.txt

```diff
@@ -105,8 +105,7 @@ salttesting==2017.6.1
 scp==0.13.2               # via junos-eznc
 setproctitle==1.1.10
 setuptools-scm==3.2.0
-singledispatch==3.4.0.3
```

Comment:
The title of this PR says it's not needed for < 3.4 - is that pointing the wrong way?

Reply:
Yeah, the commit message was wrong. Fixed.